### PR TITLE
add require("fs") in  README example

### PR DESCRIPTION
**Pull request by adam187**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Setup your aws.json file
 Then, use it in your `gulpfile.js`:
 ```javascript
 var s3 = require("gulp-s3");
+var fs = require("fs");
```
Comment (cworsley4) [inline README.md]: Whats wrong with require('./aws.json');?

**Comment by adam187** [comment]
@cworsley4 `require()` is for module loading and `aws.json` is plain json file.

**Comment by cworsley4** [comment]
Require doesn't care about that. It will read the file and parse the json for you.

**Comment by adam187** [comment]
@cworsley4 Interesting, i didn't knew that `require()` in node can do that. 
In spite of that this example isn't Mine. I only added missing module loading to complete example and make it workable.

**Comment by cworsley4** [comment]
@adam187 Ah, I see. I submitted a PR today that corrected that. We'll see if it gets accepted.

**Comment by gravitypersists** [comment]
Might want to include `var gulp = require('gulp')` while at it?

**Comment by cworsley4** [comment]
So the changes in this PR are in my fork of this repo. The repo is
abandoned. Check out... https://github.com/cworsley4/gulp-s3-ls.

**Pull request closed without merging**